library/src/main/java/com/novoda/downloadmanager/lib/Request.java

```diff
@@ -234,6 +235,17 @@ public Request alwaysAttemptResume() {
     }
 
     /**
+     * Automatically pause the download when reaching the end of the file instead of writing the last bytes to disk.
+     * This allows for a resume of the download against an updated tar file.
+     * If used this also sets {@code alwaysAttemptResume()}
+     */
+    public Request allowTarUpdates() {
+        allowTarUpdates = true;
+        alwaysAttemptResume();
```

Comment:
Should this be set as well? I'm guessing the etag of a TAR changes if the TAR is updated?